composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php" : "^5.6 || ^7.0",
-        "doctrine/dbal": ">=2.3,<2.7-dev"
+        "doctrine/dbal": ">=2.3"
```

Comment:
An open ended constraint is not a good thing, see https://getcomposer.org/doc/faqs/why-are-unbound-version-constraints-a-bad-idea.md

Since Doctrine DBAL seems to be following semver, we should probably move to a `^2.3` constraint.